### PR TITLE
NEW: Added Norwegian no_NO bank provider

### PR DESCRIPTION
**Pull request by cloveras**
11 digits. 

### What does this changes

Norwegian bank accounts have 11 digits: https://www.ecbs.org/iban/norway-bank-account-number.html

1-4: Bank identifier
5-10: Account identifier
11: Control digit

### What was wrong

Even with no_NO locale, calls to fake.bban() returned en_GB account numbers.

### How this fixes it

Returns 11 digits. No check for bank identifier (first 4 digits) or control digit (last digit).

### CODE REVIEW

File: faker/providers/bank/no_NO/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from .. import Provider as BankProvider
+
+class Provider(BankProvider):
+    bban_format = '###########'
+country_code = 'NO'
```
Comment (fcurella) [inline faker/providers/bank/no_NO/__init__.py]: this is missing indentation

Comment (cloveras) [reply faker/providers/bank/no_NO/__init__.py]: Ouch! Sorry. Fixed now.

**Comment by cloveras** [comment]
Will try!

**Comment by tommythorsen** [comment]
Hi @fcurella! I added a unit test, which seems to have made the checks happier. Do you think this PR is mergeable now?

**Comment by tommythorsen** [comment]
@cloveras Yeah, that was a silly copy-paste error. I fixed with my latest commit.

**Comment by fcurella** [comment]
Looks good! thank you! ✨

**Pull request merged**